mico-admin/src/app/service-detail-overview/service-detail-overview.component.ts

```diff
@@ -80,7 +83,7 @@ export class ServiceDetailOverviewComponent implements OnChanges, OnDestroy {
         // TODO take care of the version
         if (this.oldShortName === this.shortName) {
             if (this.oldVersion === this.version) {
-                console.log('return', this.oldShortName, this.shortName, this.oldVersion, this.version);
+                // we are on the same service (and version) as before
```

Comment:
use changes parameter of ngOnChanges function to compare with previous value. (Example in mico-form)

Reply:
the whole check with the previous version/shortName can be omitted, since onChange is only triggered if there is an actual change